js/main.js

```diff
@@ -0,0 +1,9 @@
+function getRandomCoordinate (min, max, decimal) {
+  if (min < 0 || max < 0 || decimal < 0 || max <= min) {
+    return NaN;
+  }
```

Comment:
if for и тд лучше отделять пустой строкой от последующего кода так будет легче считывать программу
```suggestion
  if (min < 0 || max < 0 || decimal < 0 || max <= min) {
    return NaN;
  }
  
```

Reply:
линтер ругается